qa-tests-backend/src/main/groovy/services/ProcessesListeningOnPortsService.groovy

```diff
@@ -15,15 +18,22 @@ class ProcessesListeningOnPortsService extends BaseService {
     }
 
     static GetProcessesListeningOnPortsResponse getProcessesListeningOnPortsResponse(
```

Comment:
It's always wrapped with retry how about adding `@Retry` annotation here? 
- https://github.com/stackrox/stackrox/pull/15194

Reply:
Done. I have also removed usage of `evaluateWithRetry`. I am not sure if that was your intention.